go/roothash/api/api.go

```diff
@@ -33,6 +33,9 @@ const (
 	LogEventRoundFailed = "roothash/round_failed"
 	// LogEventMessageUnsat is a log event value that signals a roothash message was not satisfactory.
 	LogEventMessageUnsat = "roothash/message_unsat"
+	// LogEventRoothashRuntimeReindexing is a log event value that singals a roothash runtime
+	// reindexing was run.
+	LogEventRoothashRuntimeReindexing = "roothash/runtime_reindexing"
```

Comment:
Since this is already in the roothash api package, maybe just call it "LogEventHistoryReindexing"?
```suggestion
	LogEventHistoryReindexing = "roothash/history_reindexing"
```